core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerImpl.java

```diff
@@ -51,15 +51,19 @@
     public static AndroidMediaPlayerImpl newInstance(Context context) {
         AndroidMediaPlayerFacade androidMediaPlayer = AndroidMediaPlayerFacade.newInstance(context);
         LoadTimeout loadTimeout = new LoadTimeout(new SystemClock(), new Handler(Looper.getMainLooper()));
-        return new AndroidMediaPlayerImpl(androidMediaPlayer, new MediaPlayerForwarder(), loadTimeout);
+        Heart heart = Heart.newInstance();
+        return new AndroidMediaPlayerImpl(androidMediaPlayer, new MediaPlayerForwarder(), loadTimeout, heart);
```

Comment:
Pulling out the `Heart` so that this class can be tested in a similar way to `ExoPlayerTwoImpl`